client/assets/less/pages/places.less

```diff
@@ -249,7 +240,89 @@
     width: 100%;
     @media (max-width: 767px) {
       .leaflet-control-zoom {
-        display: none;
+        top: 50px;
+      }
+    }
+  }
+}
+
+// Icons for popular places and tracks in location
+.popular-places-wrapper, .popular-tracks-wrapper {
+  display: none;
+  position: absolute;
+  z-index: 2;
+  top: 0;
+  right: 0;
+  padding-top: 115px;
+  background-color: #f8f8f8;
+  transition: all linear 0.5s;
+  width: 25%;
+  height: 100%;
+  &.ng-hide {
+    opacity: 0;
+  }
+  a:first-child {
+    color: @brand-dark;
+    display: block;
+    font-size: 24px;
+    position: absolute;
+    left: 5px;
+  }
+  .popular-div-style {
+    overflow: auto;
+    width: 100%;
+    height: 100%;
+    padding: 5px;
+    text-align: center;
+    color: @brand-dark;
+  }
+  @media (max-width: 767px) {
+    padding-top: 160px;
+  }
+  @media (max-width: 1170px) {
+    width: 320px;
+  }
+
+  // Popular Tracks in location - Styles for popular-tracks.template
+  .popular-in-location-wrapper {
+    position: relative;
+    width: 100%;
+    margin-bottom: 15px;
+    max-height: 200px;
+    min-height: 160px;
+    overflow: hidden;
+    img {
+      width: 100%;
+      height: auto;
+    }
+    .popular-in-location-gradient {
+      position: absolute;
+      top: 0;
+      padding-left: 50%;
+      width: 100%;
+      height: 100%;
+      background: linear-gradient(to right, rgba(0, 0, 0, 0) 0%, rgba(0, 0, 0, 0.65) 100%);
+    }
+    .shared-style {
+      padding: 0 4px;
+      font-weight: bold;
+      color: @brand-light;
+      position: absolute;
+    }
+    .popular-in-location-name {
+      .shared-style;
+      font-size: 18px;
+      top: 4px;
+      right: 4px;
+    }
+    .popular-in-location-address {
+      .shared-style;
+      font-size: 12px;
+      bottom: 4px;
+      right: 4px;
+      text-align: right;
+      img {
```

Comment:
Yes, because in future there will be different icons from places and tracks lookup tables